src/style.css

```
@@ -0,0 +1,42 @@
+/* Reset browser's default margin */
+.body {
+  margin: 0;
+}
+/* stars block in column */
+.stars {
+  display: flex;
+}
+
+/* each star decoration without active stars */
+.stars__star {
+  width: 16px;
+  height: 16px;
+
+  margin-right: 4px;
```

```suggestion
  margin-right: 4px;
```
And add margin-right: 0 for the last star in a line (:last-child for class stars__star)